src/DynamoCoreWpf/Views/Core/NodeView.xaml.cs

```diff
@@ -413,7 +413,7 @@ private void OnNodeViewMouseEnter(object sender, MouseEventArgs e)
             // Always set old ZIndex to the last value, even if mouse is not over the node.
             oldZIndex = NodeViewModel.StaticZIndex;
 
-            if (!previewEnabled) return; // Preview is hidden. There is no need run further.
+            if (!previewEnabled || !ViewModel.IsPreviewInsetVisible) return; // Preview is hidden. There is no need run further.
```

Comment:
@aosyatnik - we are seeing crashes with custom nodes and sliders in the latest builds. Could this be related? Let me know